CONTRIBUTING.md

```diff
@@ -163,123 +140,62 @@ description
 footer
 ```
 
-1. *label* is one of the following:
-    - `fix` - GuardianTest bug fixes.
-    - `feat` - GuardianTest features.
-    - `docs` - changes to docs, e.g. `docs(api.md): ..` to change documentation.
-    - `test` - changes to GuardianTest tests infrastructure.
-    - `devops` - build-related work, e.g. CI related patches and general changes to the browser build infrastructure
-    - `chore` - everything that doesn't fall under previous categories
-2. *namespace* is put in parenthesis after label and is optional. Must be lowercase.
-3. *title* is a brief summary of changes.
-4. *description* is **optional**, new-line separated from title and is in present tense.
-5. *footer* is **optional**, new-line separated from *description* and contains "fixes" / "references" attribution to github issues.
+1. _label_ is one of the following:
+   - `fix` - GuardianTest bug fixes.
+   - `feat` - GuardianTest features.
+   - `docs` - changes to docs, e.g. `docs(api.md): ..` to change documentation.
+   - `test` - changes to GuardianTest tests infrastructure.
+   - `devops` - build-related work, e.g. CI related patches and general changes to the browser build infrastructure
+   - `chore` - everything that doesn't fall under previous categories
+2. _namespace_ is put in parenthesis after label and is optional. Must be lowercase.
+3. _title_ is a brief summary of changes.
+4. _description_ is **optional**, new-line separated from title and is in present tense.
+5. _footer_ is **optional**, new-line separated from _description_ and contains "fixes" / "references" attribution to github issues.
 
 Example:
 
 ```
-fix(firefox): make sure session cookies work
+chore(mocking): improve consistency of allowance mocking
 
-This patch fixes session cookies in the firefox browser.
+This patch helps reduce potential issues around value collisions when mocking allowance values.
 
 Fixes #123, fixes #234
 ```
 
 ### Adding New Dependencies
 
 For all dependencies (both installation and development):
+
 - **Do not add** a dependency if the desired functionality is easily implementable.
 - If adding a dependency, it should be well-maintained and trustworthy.
 
 A barrier for introducing new installation dependencies is especially high:
+
 - **Do not add** installation dependency unless it's critical to project success.
 
 ### Running and Writing Tests
 
 - Every feature should be accompanied by a test.
 - Every public api event/method should be accompanied by a test.
-- Tests should be *hermetic*. Tests should not depend on external services.
+- Tests should be _hermetic_. Tests should not depend on external services.
 - Tests should work on all three platforms: Mac, Linux and Windows. This is especially important for screenshot tests.
-
-Playwright tests are located in [`tests`] (INSERT LINK) and use `@playwright/test` test runner.
-These are integration tests, making sure public API methods and events work as expected.
+- Tests are currently written with jest
```

Comment:
@0xLienid should we link to where / how jest can be installed in case someone doesn't already have it? Just to make it easier on people. https://jestjs.io/docs/getting-started